crates/wysiwyg/src/dom/nodes/container_node.rs

```diff
@@ -376,7 +376,7 @@ where
 
     pub fn is_empty_list_item(&self) -> bool {
         match self.kind {
-            ContainerNodeKind::ListItem => self.is_empty(),
+            ContainerNodeKind::ListItem => self.text_len() == 0,
```

Comment:
I wonder if this would be better as a change to the implementation of `is_empty()`? Perhaps that function becomes a check of if it has no children or if it has children but a text length of zero, return true.

It could be that other places check the `ListItem` using `.is_empty()` and I presume we'd want them to be using the same check

Reply:
That's a fair point, I'll check that, and try to do something a bit smart, because `text_len()` will be slower (hence why I didn't really wanted to apply these everywhere), we probably want to return as soon as we find a non-empty text node.

Reply:
I actually removed this method since it was unnecessarily checking ContainerNode type on context where it is already known it is a list item.
I added a method to check if a Container or any node actually contains text, this allows to make a difference between `really empty` and `contains no text` which are not the same.